bindsnet/learning/reward.py

```diff
@@ -54,16 +51,22 @@ def __init__(self) -> None:
         Constructor for EMA reward prediction error.
         """
         super().__init__()
+        self.rewards_predict_episode = []
 
-    def compute(self, reward: Union[float, torch.Tensor], **kwargs) -> torch.Tensor:
+    def compute(self, **kwargs) -> torch.Tensor:
         # language=rst
         """
         Computes the reward prediction error using EMA.
 
-        :param reward: Current reward.
+        Keyword arguments:
+
+        :param Union[float, torch.Tensor] reward: Current reward.
         :return: Reward prediction error.
         """
-        return super().compute(reward, **kwargs)
+        # Get keyword arguments.
+        reward = kwargs['reward']
```

Comment:
Ok, I actually was thinking to do it analogously to the way it's done in the learning rules: https://github.com/Hananel-Hazan/bindsnet/blob/c0fab385d93d9b42e3619a5bfcf43efafc320778/bindsnet/learning/__init__.py#L413
because the `KeyError` in itself is already pretty informative?

Reply:
Oh, that's right. Never mind!